src/vos/tests/vos_tests.c

```diff
@@ -21,6 +21,7 @@
 #include <getopt.h>
 #include <daos_srv/vos.h>
 #include <vos_internal.h>
+#include <linux/limits.h>
```

Comment:
why do we include linux/limits.h if we define our own max?   Can we use PATH_MAX - (some number of bytes for file name) instead?

Reply:
I previously used PATH_MAX and changed to a short string to avoid compiling issue on some OS in CI. Now limits.h is not needed. I just removed this line and pushed to git.